howto/use-w3c-tracecontext/README.md

```diff
@@ -0,0 +1,243 @@
+# W3C Trace Context - Background
+
+Distributed tracing is a methodology implemented by tracing tools to follow, analyze and debug a transaction across multiple software components. Typically, a distributed trace traverses more than one component which requires it to be uniquely identifiable across all participating systems. Trace context propagation passes along this unique identification. Today, trace context propagation is implemented individually by each tracing vendor. In multi-vendor environments, this causes interoperability problems, like:
```

Comment:
This article has several sections, please add a "contents" section with links to the sub headers for easy navigation

Reply:
done.